src/components/sections/Section.jsx

```diff
@@ -63,7 +63,9 @@ export default class Section extends React.Component {
             panes.push(
                 <Pane ref="pane0" key={ subSections[0].path }
                     onOpenPane={ this.onOpenPane.bind(this, 0) }
-                    onPushPane={ this.onPushPane.bind(this) }/>
+                    onPushPane={ this.onPushPane.bind(this) }
+                    onReplace={ this.onReplacePane.bind(this, 0) }
```

Comment:
Replace the first pane? Where is this supposed to be used?

Reply:
This happens (via `gotoPane()`) when you select a view and the url changes from `views` to `views:1`.

See `PersonViewsPane.js:223`:

```jsx
<li key={ viewItem.data.id }
    className="PersonViewsPane-viewItem"
    onClick={ () => this.gotoPane('views', viewItem.data.id) }
    >
    <div className="PersonViewsPane-viewItemIcon"/>
    <span className="PersonViewsPane-viewItemTitle">{ viewItem.data.title }</span>
</li>
```